code/modules/hydroponics/seeds.dm

```diff
@@ -28,111 +28,143 @@
 
 
 /obj/item/seeds/poppyseed
+	name = "Poppy Seed"
```

Comment:
these are all improper nouns

Reply:
Fucking how, how does making them lower case make or break them being improper nouns? I need an explanation.

Reply:
And if anything they should be capitalized, because it is the name of a thing